src/metadata.cc

```diff
@@ -81,6 +81,22 @@ class MetadataWorker : public Nan::AsyncWorker {
         memcpy(baton->icc, icc, iccLength);
         baton->iccLength = iccLength;
       }
+      // IPTC
+      if (image.get_typeof(VIPS_META_IPCT_NAME) == VIPS_TYPE_BLOB) {
+        size_t iptcLength;
+        void const *iptc = image.get_blob(VIPS_META_ICC_NAME, &iptcLength);
```

Comment:
```diff
-        void const *iptc = image.get_blob(VIPS_META_ICC_NAME, &iptcLength);
+        void const *iptc = image.get_blob(VIPS_META_IPTC_NAME, &iptcLength);
```

Reply:
@lovell fixed. Tests passing locally. Waiting for CI